EIPS/eip-3171.md

```diff
@@ -0,0 +1,148 @@
+---
+eip: 3171
+title: CREATE3
+author: Moody Salem <moody@uniswap.org>
+category: Core
+type: Standards Track
+status: Draft
+created: 2020-12-22
+---
+
```

Comment:
Where should discussions for this EIP go?

Reply:
@moodysalem still missing a `discussions-to` section. Merging is blocked until this is resolved.